pkg/loader/bundle/bundle.go

```diff
@@ -170,24 +171,24 @@ func loadPorts(service Service) ([]kobject.Ports, string) {
 			Protocol:      p,
 		})
 	}
-	return ports, ""
+	return ports, nil
 }
 
 // LoadFile loads dab file into KomposeObject
-func (b *Bundle) LoadFile(files []string) kobject.KomposeObject {
+func (b *Bundle) LoadFile(files []string) (kobject.KomposeObject, error) {
 	komposeObject := kobject.KomposeObject{
 		ServiceConfigs: make(map[string]kobject.ServiceConfig),
 		LoadedFrom:     "bundle",
 	}
 	file := files[0]
 	buf, err := ioutil.ReadFile(file)
 	if err != nil {
-		log.Fatalf("Failed to read bundles file: %s ", err)
+		return kobject.KomposeObject{}, errors.Wrap(err, "ioutil.ReadFile failed, Failed to read bundles file")
```

Comment:
Also follow up question here you are using the error message as `ioutil.ReadFile` failed which is kinda absolute name but below at [line#191](https://github.com/kubernetes-incubator/kompose/pull/462/files#diff-fe05d91dde638f14ac7bf8afa193be77R191) you have just used the name which is `loadFile`, why is that so?